barretenberg/cpp/src/barretenberg/polynomials/polynomial.test.cpp

```diff
@@ -74,74 +74,74 @@ TEST(Polynomial, AddScaledEdgeConditions)
     // Suppress warnings about fork(), we're OK with the edge cases.
     GTEST_FLAG_SET(death_test_style, "threadsafe");
     using FF = bb::fr;
-    auto test_subset_good = []() {
```

Comment:
had to update these tests because we allow for some of these operations where the active ranges are different.